.github/workflows/rspec-test.yml

```diff
@@ -67,10 +67,20 @@ jobs:
               npm run test
           fi
 
-      - name: Test
-        if: ${{ steps.secret-check.outputs.available != 'true' }}
+      - name: Run RSpec to Output File
```

Comment:
Slight problem here. Most of our projects have the secret to send code coverage to codeclimate. By removing that check, we will be running our tests twice, which will add quite a bit of time to the execution of this action.

Reply:
I suppose we could split this out into another workflow entirely and have it run in parallel? Not in love with the duplication of that though.

Reply:
Added test back, no longer running rspec twice